src/components/Footer/Footer.js

```diff
@@ -1,17 +1,44 @@
 import React from "react";
 import styles from "../Footer/Footer.module.css";
 import PropTypes from "prop-types";
+import classnames from "classnames";
 
-const Footer = ({ count }) => (
+const Footer = ({
+  count_all,
+  count_end,
+  count_inWork,
+  sorting,
+  onClickSort,
+}) => (
   <div className={styles.footer}>
-    <div>
-      <span>Осталось дел: {count}</span>
-    </div>
     <div className={styles.button__wrap}>
-      <button className={styles.button}>Все</button>
-      <button className={styles.button}>Активные</button>
-      <button className={styles.button}>Завершённые</button>
-      <button className={styles.button}>Удалить выполненные</button>
+      <button
+        className={classnames({
+          [styles.button]: true,
+          [styles.button_active]: sorting === "Завершенные",
+        })}
+        onClick={() => onClickSort("Завершенные")}
+      >
+        Завершённые<p className={styles.count_item}>{count_end}</p>
+      </button>
+      <button
+        className={classnames({
+          [styles.button]: true,
+          [styles.button_active]: sorting === "Незавершенные",
+        })}
+        onClick={() => onClickSort("Незавершенные")}
+      >
+        Незавершённые<p className={styles.count_item}>{count_inWork}</p>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'count_inWork' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)